### DISCUSSION TITLE
[PyNE] Contratação de RPA

### DISCUSSION THREAD

**Discussion by cecivieira** [apyb/comunidade#93] [Geral]
Olá,

Precisamos contratar 2 pessoas para auxiliar no evento nas áreas de limpeza e técnico sonoro. Gostaria de informações como realizar isso via RPA (Recibo de Pagamento ao Autônomo).

**Comment by muriloviana** [comment]
@cecivieira precisamos dos seguintes dados: nome completo, endereço, documentos RG/CPF, nº do PIS.

**Comment by cecivieira** [comment]
de @muriloviana :

Sobre o RPA, resposta da contabilidade:
"""
Pode sim, lembrando que no pagamento deve ser descontado 11% de INSS e se for o caso IRRF conforme tabela. 
Qualquer dúvida estou a disposição. 
"""
